packages/fxa-auth-server/lib/payments/stripe.ts

```diff
@@ -2722,7 +2721,7 @@ export class StripeHelper {
       productNameOld,
       productIconURLOld,
       productDownloadURLOld,
-      productPaymentCycleOld,
+      productPaymentCycleOld: planOld.interval ? planOld.interval : baseDetails.productPaymentCycleNew,
```

Comment:
fyi in case you don't know about the nullish coalescing operator: the above can be shortened to `planOld.interval ?? baseDetails.productPaymentCycleNew`

Reply:
I did not, NICE! I'll update to use it. Thanks!